funnel/forms/profile.py

```diff
@@ -49,6 +49,10 @@ class ProfileForm(OrganizationForm):
         filters=[forms.filters.none_if_empty()],
     )
 
+    def set_queries(self):
+        if self.edit_obj:
+            self.logo_url.profile = self.edit_obj.name
```

Comment:
This method should not be necessary. Imgee's popup gallery should work without specifying a profile, and should pick it from the current user.